lib/testing_appserver/modules/documents/modules/pop_up_windows/documents_sharing_settings.rb

```diff
@@ -11,16 +11,18 @@ module DocumentsSharingSettings
     div(:plus_share, xpath: "//div[contains(@class, 'sharing_panel-plus-icon')]")
     div(:add_users, xpath: "//div[@label='Add users']") # add_id
     div(:add_groups, xpath: "//div[@label='Add groups']") # add_id
-    button(:save_sharing, xpath: "//button[contains(@class, 'sharing_panel-button')]")
+    button(:save_sharing, xpath: "//button[contains(@class, 'sharing_panel-button')
```

Comment:
I think string like that are better to be split like this
```
element(:enable_external_link, xpath: "//label[contains(@class, 'sharing-row__toggle-button')"\
                                      "or contains(@class, 'external-link__toggler')]")
```

It's harder to mess something wrong with them if they are separated by `\` symbol